src/BalconyBotanica/Adapters/Database/DummyDatabase/DummyPlantData.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BalconyBotanica.Adapters.Database.DummyDatabase
+{
+    public class DummyDatabase
+    {
+
+        readonly PlantDataDbo plantData1 = new()
+        {
+            id = "1",
+            common_name = "Sunflower",
+            scientific_name = ["Helianthus annuus"],
+            other_name = ["Helianthus", "Common Sunflower"],
+            cycle = "Annual",
+            watering = "FREQUENT",
+            sunlight = ["FULL_SUN"]
```

Comment:
Can you write this in C#? Then there is type-safety and typos will be caught by the compiler:

```suggestion
            sunlight = [Sunlight.FULL_SUN.toString()]
```

Same applies to the other enums like cycle and watering etc.

Reply:
Jup! Sunlight.FULL_SUN.ToString()
Good one.